insights/parsers/grub2_editenv_list.py

```diff
@@ -0,0 +1,80 @@
+"""
+Grub2EditenvList - command ``grub2-editenv list``
+=================================================
+
+This parser reads the output of command ``grub2-editenv list``, which list the
+current variables in GRUB environment block file. The output without error message
+is laid out in a key=value format similar to an ini file except it doesn't have any
+laid headers.
+"""
+
+from insights import get_active_lines, parser
+from insights.parsers import SkipException
+from insights.parsers.grubenv import GrubEnv
```

Comment:
@TZ3070 Since this is importing from the grubenv parser file, could you move this parser to that file instead of having a separate file just for this parser since they're both about grubenv.

Reply:
Updated